patches/arcade/0013-Add-reference-to-PlatformAbstractions.patch

```diff
@@ -0,0 +1,24 @@
+From 75487513ada7c1dafaeb7aff7f287f9c178e4623 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Wed, 1 Apr 2020 16:32:18 +0000
+Subject: [PATCH] Add reference to PlatformAbstractions
```

Comment:
Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk.csproj doesn't have a reference to PlatformAbstractions.  It builds in the production build because it has a reference to Microsoft.Extensions.DependencyModel, which also brought in PlatformAbstractions.  That dependency has been removed with https://github.com/dotnet/core-setup/pull/5218.  The tarball build uses a newer version of Microsoft.Extensions.DependencyModel with the dependency removed, so an explicit dependency needs to be added now until Arcade removes their dependency on PlatormAbstractions.

Reply:
Thanks. Is it reasonable to put this in the patch and/or link to an open issue?

Reply:
I added the text above to the patch.